op-chain-ops/cmd/celo-migrate/non-ancients.go

```diff
@@ -75,11 +75,18 @@ func migrateNonAncientsDb(newDB ethdb.Database, lastBlock, numAncients, batchSiz
 		}
 	}
 
+	prevBlockNumber := uint64(numAncients - 1)
+
 	for i := numAncients; i <= lastBlock; i += batchSize {
 		numbersHash := rawdb.ReadAllHashesInRange(newDB, i, i+batchSize-1)
 
 		log.Info("Processing Block Range", "process", "non-ancients", "from", i, "to(inclusve)", i+batchSize-1, "count", len(numbersHash))
 		for _, numberHash := range numbersHash {
+			if numberHash.Number != 0 && numberHash.Number != prevBlockNumber+1 {
```

Comment:
This check for zero is also redundant
```suggestion
			if numberHash.Number != prevBlockNumber+1 {
```